.github/workflows/ci-mobile-save-for-later-user-deletion.yaml

```
@@ -20,6 +20,9 @@ jobs:
           distribution: temurin
           cache: sbt
 
+      - name: Run Tests
+        run: sbt "project mobile-save-for-later-user-deletion" test
+
       - name: Create artifacts
         run: sbt "project mobile-save-for-later-user-deletion" assembly
```

Ah, so `assembly` doesn't running the tests, so the tests weren't getting run at all?!? Good spot!

Arguably, for a faster build, we can avoid initialising `sbt` twice, and just run both `test` & `assembly` in a single step:

```suggestion
      - name: Run Tests & Create Artifacts
        run: sbt "project mobile-save-for-later-user-deletion" test assembly
```